google/cloud/pubsub/message.h

```diff
@@ -294,6 +300,15 @@ inline void SetAttribute(std::string const& key, std::string value,
   (*m.proto_.mutable_attributes())[key] = std::move(value);
 }
 
+inline std::string GetAttributeValue(std::string const& key,
+                                     pubsub::Message& m) {
+  auto value = m.proto_.attributes().find(key);
+  if (value != m.proto_.attributes().end()) {
+    return value->second;
+  }
+  return "";
```

Comment:
I guess this means there is no need to distinguish between an empty and a missing value.  If so, (nit warning) you can probably save a byte by spelling this as `std::string{}`.  Woo!